cloudos_cli/clos.py

```diff
@@ -552,6 +552,8 @@ def get_job_list(self, workspace_id, last_n_jobs=30, page=1, archived=False,
             very large int or 'all' to get all user's jobs.
         page : int
             Response page to get (ignored when using filters - starts from page 1).
+        page_size : int
+            Number of jobs to retrieve per page (ignored when using filters - always 10).
```

Comment:
For this (and the above --page), it says that it is ignored when filters are used. However, in the examples in README.md it seems they do take effect when filters are applied. Please, add the correct explanation.